### DISCUSSION TITLE
Show Directories / Folders / Dirs on top, or first ?

### DISCUSSION THREAD

**Discussion by jult** [DirectoryLister/DirectoryLister#1296] [Ideas & Features]
Really surprised this is not the case by default. All other indexers do this, put the dirs on top of file listings. Apache does it, nginx does it, IIS does it.

**Comment by PHLAK** [comment]
This is the default behavior (see the [demo](https://demo.directorylister.com/)). Make sure the [sort order](https://docs.directorylister.com/configuration/app-config-reference#sort_order) is set to `type`.
